### PR TITLE
Updated language replace

### PR DESCRIPTION
**Pull request by JorgWieme**
http://localhost:3030/something_ending_on_en/en/ becomes http://localhost:3030/something_ending_on_nl/nl/  when trying to change to change languages. The new replace has this fixed.

### CODE REVIEW

File: view/topbar.twig [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <div id="language"><span class="navigation-font">|</span>
   {% for langcode, langdata in languages %}
   {% if request.lang != langcode %}
-  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("#{request.lang}/"): "#{langcode}/"}) }}"> {{langdata.name}}</a>
+  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("/#{request.lang}/"): "/#{langcode}/"}) }}"> {{langdata.lemma}}</a>
```
Comment (osma) [inline view/topbar.twig]: Is it intentional that you replace `langdata.name` with `langdata.lemma` here?

Comment (JorgWieme) [reply view/topbar.twig]: Yes, before changing it, all languages were displayed with /in_this_language**

Comment (kinow) [reply view/topbar.twig]: Interesting that when running with Docker (see pull #700), I get the same, i.e. **in_this_language**. But when I get the changes in this pull request, at least that text is fixed, and displays now "svenka" and "English" instead. So at least it seems to fix this issue for my environment.

Comment (kinow) [reply view/topbar.twig]: Fixes the text, but now I noticed that the links are not working. If I change in the URL from /fi/ to /en/, everything works. Clicking on the links doesn't seem to be working for me 👍

**Comment by osma** [comment]
Exactly, the `in_this_language` is a special gettext id that should be translated to a phrase in the target language. If you see it in the UI the most likely cause is that you have a language defined in `config.inc` but haven't installed the corresponding locale. Not sure how locales work inside Docker containers, but you should make sure all locales mentioned in `config.inc` are available also from inside the container.

**Comment by kinow** [comment]
Aaahh the locale packages. Got to update the pull request for docker then, and include a few default locales. Thanks Osma!!!

**Comment by osma** [comment]
As discussed above, the **in_this_language** problem is caused by missing locales on the system and is not relevant. `langdata.name` should be left intact, not changed to `langdata.lemma`.

The original problem is real - if the vocabulary name ends with a language tag, it will be changed by the language change links. But the suggested fix is also broken, as pointed out by @henriyli: the language change links stop working e.g. on the About page. So we cannot merge this as is.

**Comment by joelit** [comment]
It seems that this is a limitation of twig's replace() function, which does not have proper regex capability. The language replacement will have to be handled somewhere else than topbar.twig. Closing this issue and creating a new one.

**Pull request closed without merging**